config/device.go

```diff
@@ -60,11 +61,16 @@ type Board struct {
 func ParseFromFile(configPath string) (*Device, error) {
 	cfg := &Device{
 		General: General{
-			RunEvery:         time.Minute,
-			NCSToolChainBase: "~/ncs",
-			NCSVersion:       "v2.5.0",
-			Manufacturer:     "FFexix113",
-			DeviceName:       "dongle",
+			RunEvery: time.Minute,
+			NCSToolChainBase: func() string {
+				if runtime.GOOS == "windows" {
+					return "${NCS_TOOLCHAIN_BASE}"
+				}
+				return "$NCS_TOOLCHAIN_BASE"
```

Comment:
```suggestion
				if runtime.GOOS == "windows" {
					return "C:\\ncs"
				}
				return "~/ncs"
```

When resolving NCS path in `GetToochainsPath` it will get value of this environment variable, and if it is not empty - will prefer it over that from configuration file.